tools/novoalign.py

```diff
@@ -177,7 +177,7 @@ def align_one_rg_bam(self, inBam, refFasta, outBam, rgid=None, options=None, min
         tmp_sam = util.file.mkstempfname('.novoalign.sam')
         tmp_sam_err = util.file.mkstempfname('.novoalign.sam.err')
         cmd = [self.install_and_get_path(), '-f', one_rg_inBam] + list(map(str, options))
-        cmd = cmd + ['-F', 'BAMPE', '-d', self._fasta_to_idx_name(refFasta), '-o', 'SAM']
+        cmd = cmd + ['-F', 'BAM', '-d', self._fasta_to_idx_name(refFasta), '-o', 'SAM']
```

Comment:
weirdly, if you google for "Novoalign manual", there are a few top hits referring to different versions. The one for [v3.07](http://www.novocraft.com/userfiles/file/Novocraft.pdf) describes this. The one for [v3.02](http://www.novocraft.com/wp-content/uploads/Novocraft.pdf) does not.